src/game/behaviors/door.inc.c

```diff
@@ -34,7 +34,10 @@ void set_door_camera_event(void) {
 }
 
 void play_door_open_noise(void) {
-    s32 isMetalDoor = cur_obj_has_model(MODEL_HMC_METAL_DOOR);
+    s32 isMetalDoor = (
+        cur_obj_has_model(MODEL_HMC_METAL_DOOR) ||
+        cur_obj_has_model(MODEL_CASTLE_METAL_DOOR)
```

Comment:
Idk whether anybody disagrees with me on this, but this does not match vanilla behavior. Maybe should be thrown behind the vanilla checks define or something.

Reply:
males sense to me

Reply:
or remove and we can agree on a solution later since this is unrelated to everything else in the pr